src/routes/userRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import UsersController from '../controllers/UsersController';
+import TokenAuthenticate from '../helpers/TokenAuthenticate';
+import UserMiddleware from '../middlewares/UserMiddleware';
+
+const userRouter = express.Router();
+
+userRouter.post(
+  '/artists/follow/:artistId',
+  TokenAuthenticate.tokenVerify,
+  UserMiddleware.validateFollowerAndArtistID,
+  UsersController.userFollow,
+);
+
+userRouter.post(
+  '/artists/unfollow/:artistId',
+  TokenAuthenticate.tokenVerify,
```

Comment:
Please can you take `TokenAuthenticate.tokenVerify` to the index router since all routes here are protected